tests/NLog.UnitTests/LayoutRenderers/CallSiteTests.cs

```diff
@@ -31,6 +31,9 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 // 
 
+using System.IO;
+using System.Threading.Tasks;
+
```

Comment:
I would be more consistent yes. Resharper did this for me, but will fix it (cannot configure this in Resharper AFAIK)